packages/components/src/components/tabs/tabs.tsx

```diff
@@ -14,20 +14,20 @@ declare module 'react' {
 }
 
 type TTabsProps = RouteComponentProps & {
-    active_icon_color: string;
+    active_icon_color?: string;
     active_index?: number;
-    background_color: string;
+    background_color?: string;
     bottom?: boolean;
-    center: boolean;
-    children: React.ReactElement[];
+    center?: boolean;
+    children: (React.ReactElement | null)[];
     className?: string;
-    fit_content: boolean;
+    fit_content?: boolean;
     has_active_line?: boolean;
     has_bottom_line?: boolean;
     header_fit_content?: boolean;
-    history: History;
-    icon_color: string;
-    icon_size: number;
+    history?: History;
+    icon_color?: string;
+    icon_size?: number;
```

Comment:
Ensure that declaring everything as optional should have a fallback be it a condition or a default value

Reply:
I'll add default value